Bluepill-cli/BPInstanceTests/BluepillTests.m

```diff
@@ -53,6 +53,7 @@ - (void)setUp {
     self.config.junitOutput = NO;
     self.config.testRunnerAppPath = nil;
     self.config.testing_CrashAppOnLaunch = NO;
+    self.config.testing_BluepillCliTest = YES;
```

Comment:
What is the logic behind `testing_BluepillCliTest`? I.e. why is this necessary?

Reply:
+1 need more doc/description for this flag